### ISSUE TITLE
Jaws does not announce aria-valuetext in an HTML slider

### ISSUE THREAD

**Issue by Wildebrew** [FreedomScientific/VFO-standards-support#918]
I have created a slider based on `<input type="range">` with a twist (see attached).
For values 3, 8 and 11 there is an additional announcement, think of it as a slider with external labels associated with certain values.
I use Javascript to insert aria-valuetext with the value + the relevant announcement when user selects the appropriate value:
* 3  aria-valuetext="3 - quiet"
* 8 - aria-valuetext="8 - loud"
* 11 - aria-valuetext="11 - my ears bleed"

When I am navigating the slider with the arrow keys I expect to hear the aria-valuetext announced for these values. This works perfectly with both NVDA + Chrome and Voiceover + Safari, but does not work in Jaws (with Chrome or Firefox).
I have to exit forms mode with those values selected and use browse mode to see the aria-valuetext.
That is not good enough, a Jas user needs to hear any special value announcements while selecting from the slider with the arrow keys.

[sliderTest2.html](https://github.com/user-attachments/files/21803814/sliderTest2.html)

**Comment by JAWS-test** [comment]
Related: https://github.com/FreedomScientific/standards-support/issues/916

**Comment by jessegreenberg** [comment]
This seems to be fixed in the latest JAWS version 2025.2508.120.
